App/Data Sources/MessageListDataSource.swift

```diff
@@ -96,6 +96,10 @@ extension MessageListDataSource: UITableViewDataSource {
     private func viewModelForMessage(at indexPath: IndexPath) -> MessageListCell.ViewModel {
         let message = self.message(at: indexPath)
         let theme = Theme.defaultTheme()
+          
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "d MMM"
```

Comment:
Let's move this formatter and its configuration to an instance property on `MessageListDataSource`. It's surprisingly expensive to change the format.

Reply:
I've addressed this by moving it into a pre-existing sentdateformatter variable